infra/frontend/service/main.tf

```diff
@@ -30,6 +30,7 @@ locals {
   service_config                                 = local.environment_config.service_config
   database_config                                = local.environment_config.database_config
   incident_management_service_integration_config = local.environment_config.incident_management_service_integration
+  domain                                         = local.environment_config.domain
```

Comment:
Oh, just getting to this spot in the PR, this is the exact thing I was talking about above. This is the `environment_config` from `app_config`. Can we maybe remove the output from `env_config` for domain entirely?